added MAC install instructions and other doc changes. Removed data fr…

lucask07 opened the pull request:
…om docstrings. Added test_defines.v to .gitignore

Added MAC OK setup instructions to the end of the Installation page. Noted that the _tests_ requires a clone (not just the package). 

Added a link to the ADS8686 eval kit and to the DAC80508 breakout board. 

Slight modifications to Python docstrings to improve the formatting on readthedocs.

When running tests _test_defines.v_ was created so added that to the .gitignore

Ajstros commented:
@lucask07 I fixed the Mac Setup link on the installation page and corrected "ttools" to "otools," but that part is a guess. Should it be "otools?"

lucask07 commented:
> @lucask07 I fixed the Mac Setup link on the installation page and corrected "ttools" to "otools," but that part is a guess. Should it be "otools?"

Yikes, this is a double typo: _otool_ is correct

Ajstros commented:
Okay, it is "otool" in the documentation now. Merging this
